checker/tests/run-pass/promoted_slice.rs

```diff
@@ -17,94 +17,96 @@ pub fn promote_u8(value: &[u8]) -> u8 {
     const TAG: &[u8] = &[1, 2, 3];
     match value {
         TAG => 10,
-        _ => 20
+        _ => 20,
```

Comment:
Nothing to see here, just updating the formatting of a test that failed at some point.